The mouse panel needs gsd_device_manager_get() which needs GUdev.

wwan needs network-manager

Index: panels/meson.build
--- panels/meson.build.orig
+++ panels/meson.build
@@ -6,7 +6,6 @@ panels = [
   'color',
   'display',
   'keyboard',
-  'mouse',
   'multitasking',
   'notifications',
   'online-accounts',
@@ -18,7 +17,6 @@ panels = [
   'sound',
   'system',
   'universal-access',
- 'wwan',
 ]
 
 if host_is_linux
